scripts/generate_embeddings_openstack.py

```diff
@@ -53,9 +53,18 @@ def __init__(self, docs_path, base_url=ROOT_URL, version="18.0"):
         self.version = version
 
     def url_function(self, file_path: str):
-        return clean_url(
-            self.base_url.format(self.version) + "/" + str(Path(file_path).parent.name)
-        )
+        if "release-notes" in file_path:
```

Comment:
Not a big fan of sub-string comparison but, I think it should be OK

Reply:
Agreed. I considered what I'd need to refactor to further split these up and decided it wasn't worth it since the release notes are the last thing "missing" of the content from the documentation landing page. This tiny compromise saves a lot of changes.